js-library/README.md

```diff
@@ -12,6 +12,12 @@ Install library
 npm install @dfinity/verifiable-credentials
 ```
 
+The bundle needs peer dependencies, be sure that following resources are available in your project as well.
+
+```bash
```

Comment:
Note: actually, figured this out only recently, npm automatically install the peer dependencies. On the contrary, yarn doesn't.

Reply:
Yes, I was reading about it that latest npm version do so. Good to know that yarn doesn't.